apps/api/src/routers/admin.py

```diff
@@ -210,6 +210,61 @@ async def confirm_attendance() -> None:
         )
 
 
+@router.get(
+    "/attending",
+    dependencies=[
+        Depends(require_role([Role.DIRECTOR, Role.ORGANIZER, Role.VOLUNTEER]))
+    ],
+)
+async def attending() -> list[dict[str, object]]:
+    """Fetch all applicants who have a status of ATTENDING"""
+    records = await mongodb_handler.retrieve(
+        Collection.USERS,
+        {"role": Role.APPLICANT, "status": Status.ATTENDING},
+        [
+            "_id",
+            "status",
+            "role",
+            "application_data.first_name",
+            "application_data.last_name",
+        ],
+    )
+
+    return records
+
+
+@router.post("/checkin/{uid}")
+async def checkin(
+    uid: str,
+    associate: User = Depends(
+        require_role([Role.DIRECTOR, Role.ORGANIZER, Role.VOLUNTEER])
+    ),
+) -> None:
+    """Check-in applicant at IrvineHacks"""
```

Comment:
Very small grammar nitpick: Should be "Check in" instead of "check-in", since it's being used as a verb.